client/components/domains/transfer-domain-step/transfer-domain-options.jsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormLegend from 'components/forms/form-legend';
+import { getProductsList } from 'state/products-list/selectors';
+
+class TransferDomainOptions extends React.PureComponent {
+	static propTypes = {
+		domain: PropTypes.string,
+		onSubmit: PropTypes.func,
+	};
+
+	onClick = () => {
+		const { dnsImport, nameservers, privacy } = this.state;
+
+		this.props.onSubmit( this.props.domain, nameservers, privacy, dnsImport );
+	};
+
+	state = {
+		dnsImport: true,
+		nameservers: true,
+		privacy: true,
+	};
+
+	toggle( property, value ) {
+		this.setState( {
+			[ property ]: value,
+		} );
+	}
+
+	toggleNameservers = event => {
+		const value = event.target.checked;
+
+		this.toggle( 'nameservers', value );
+		if ( ! value ) {
+			this.toggle( 'dnsImport', false );
+		}
+	};
+
+	togglePrivacy = event => {
+		this.toggle( 'privacy', event.target.checked );
+	};
+
+	toggleDnsImport = event => {
+		this.toggle( 'dnsImport', event.target.checked );
+	};
+
+	render() {
+		const cost = this.props.products.private_whois
+			? this.props.products.private_whois.cost_display
+			: null;
+		const { translate } = this.props;
+
+		const headerLabel = translate(
+			"You have a few options for how to set up your domain once it's transferred. " +
+				'Review these options and choose the ones that apply to ensure that everything is ' +
+				'configured the way you want it.'
+		);
+
+		return (
+			<div className="transfer-domain-step__options">
+				<SectionHeader>{ headerLabel }</SectionHeader>
+				<Card>
+					<FormFieldset>
+						<FormLabel>
```

Comment:
Props on using form labels - it's a pet peeve of mine when you have to hunt with the mouse pointer for the checkbox, when there's a built in functionality that handles it (not to mention how it impacts a11y).